kie-drools-wb/kie-drools-wb-webapp/src/main/webapp/WEB-INF/jboss-deployment-structure.xml

```diff
@@ -51,6 +51,13 @@
       <module name="org.slf4j.ext"/>
       <module name="org.slf4j.jcl-over-slf4j"/>
 
+      <!-- Wildfly / EAP security management provider's client dependencies. -->
+      <module name="org.jboss.as.controller-client"/>
+      <module name="org.jboss.as.domain-management"/>
```

Comment:
According to https://access.redhat.com/articles/1122333, the `domain-management` module is private. This means that there is _no_ backwards compatibility guarantee from the WF/EAP. If we really need to depend on this module, it should be moved few lines below to the private/unsupported section (as the comment suggests).